Tests/Creation/Frame/Content/String/ID3FrameFromURLStringContentCreatorTest.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ID3FrameFromStringContentCreatorTest.swift
+//
+//  Created by Nolaine Crusher on 03/31/2020.
+//  2018 Fabrizio Duroni.
+//
+
+import XCTest
+@testable import ID3TagEditor
+
+class ID3FrameFromURLStringContentCreatorTest: XCTestCase {
```

Comment:
Ditto here, though I haven't gotten as far. I'm trying to model these tests after what you did with `ID3CommentTypesFrameCreatorTest` but...?